cmd/internal/cli/build.go

```diff
@@ -265,7 +265,17 @@ func checkBuildTarget(path string) error {
 			return fmt.Errorf("only sandbox update is supported: %s is not a directory", path)
 		}
 		if !buildArgs.update && !forceOverwrite {
-			question := "Build target already exists. Do you want to overwrite? [N/y] "
+
+			var question string
```

Comment:
Apologies - I should have made this comment earlier. Please let me know if you'd rather I commit this change to the branch, rather than you needing to do it, and I can take care of that to merge this.


It would probably be more common to write something like this...

```
question := fmt.Sprintf("Build target '%s' already exists. Do you want to overwrite? [N/y]", f.Name())

if isDefFile, _ := parser.IsValidDefinition(path); isDefFile {
    question = fmt.Sprintf("Build target '%s' is a definition file that will be overwritten. Do you still want to overwrite? [N/y]", f.Name())	
}
```

The pattern of setting a default value with `:=` and overriding for the special case, rather than using `var` and `if/else` blocks is generally used in our code, and in Go code in general.

Reply:
I have fixed it on my side with the new commit. Thanks for the advice, I'll make sure to conform to that style in future.